### ISSUE TITLE
[Bug]: has failed to load correctly

### ISSUE THREAD

**Issue by krjencik** [Cheos137/ArmorpointsPlusplus#36]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Minecraft wont run

### Version

3.0.4 for Minecraft Forge 1.19

### Version

forge-1.19-41.0.98

### To Reproduce

1. download forge 41.0.98
2. download this mod
3. run Minecraft

### Expexted Behavior

See the Minecraft's main menu

### Relevant Log / Crash-Report Output

```shell
[15Jul2022 09:45:37.519] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, krjencik, --version, 1.19-forge-41.0.98, --gameDir, C:\Users\Kr Jen\AppData\Roaming\.minecraft, --assetsDir, C:\Users\Kr Jen\AppData\Roaming\.minecraft\assets, --assetIndex, 1.19, --uuid, f3c292de32e141edb99650e908dd6eb5, --accessToken, ????????, --clientId, MzU2YzllYWQtNDEwYi00N2M3LWI1MTYtZTI2NWQ2MmJkMmI4, --xuid, 2535429182244007, --userType, msa, --versionType, release, --launchTarget, forgeclient, --fml.forgeVersion, 41.0.98, --fml.mcVersion, 1.19, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220607.102129]
[15Jul2022 09:45:37.535] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 10.0.8+10.0.8+main.0ef7e830 starting: java version 17.0.3 by Microsoft; OS Windows 10 arch amd64 version 10.0
[15Jul2022 09:45:38.238] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/C:/Users/Kr%20Jen/AppData/Roaming/.minecraft/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2390!/ Service=ModLauncher Env=CLIENT
[15Jul2022 09:45:39.004] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\Kr Jen\AppData\Roaming\.minecraft\libraries\net\minecraftforge\fmlcore\1.19-41.0.98\fmlcore-1.19-41.0.98.jar is missing mods.toml file
[15Jul2022 09:45:39.004] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\Kr Jen\AppData\Roaming\.minecraft\libraries\net\minecraftforge\javafmllanguage\1.19-41.0.98\javafmllanguage-1.19-41.0.98.jar is missing mods.toml file
[15Jul2022 09:45:39.019] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\Kr Jen\AppData\Roaming\.minecraft\libraries\net\minecraftforge\lowcodelanguage\1.19-41.0.98\lowcodelanguage-1.19-41.0.98.jar is missing mods.toml file
[15Jul2022 09:45:39.019] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\Kr Jen\AppData\Roaming\.minecraft\libraries\net\minecraftforge\mclanguage\1.19-41.0.98\mclanguage-1.19-41.0.98.jar is missing mods.toml file
[15Jul2022 09:45:39.160] [main/INFO] [net.minecraftforge.fml.loading.moddiscovery.JarInJarDependencyLocator/]: No dependencies to load found. Skipping!
[15Jul2022 09:45:42.323] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
[15Jul2022 09:45:42.370] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeclient' with arguments [--version, 1.19-forge-41.0.98, --gameDir, C:\Users\Kr Jen\AppData\Roaming\.minecraft, --assetsDir, C:\Users\Kr Jen\AppData\Roaming\.minecraft\assets, --uuid, f3c292de32e141edb99650e908dd6eb5, --username, krjencik, --assetIndex, 1.19, --accessToken, ????????, --clientId, MzU2YzllYWQtNDEwYi00N2M3LWI1MTYtZTI2NWQ2MmJkMmI4, --xuid, 2535429182244007, --userType, msa, --versionType, release]
[15Jul2022 09:45:49.823] [pool-3-thread-1/INFO] [net.minecraft.util.datafix.DataFixers/]: Building unoptimized datafixer
[15Jul2022 09:45:53.132] [Render thread/WARN] [net.minecraft.server.packs.VanillaPackResources/]: Assets URL 'union:/C:/Users/Kr%20Jen/AppData/Roaming/.minecraft/libraries/net/minecraft/client/1.19-20220607.102129/client-1.19-20220607.102129-srg.jar%23173!/assets/.mcassetsroot' uses unexpected schema
[15Jul2022 09:45:53.132] [Render thread/WARN] [net.minecraft.server.packs.VanillaPackResources/]: Assets URL 'union:/C:/Users/Kr%20Jen/AppData/Roaming/.minecraft/libraries/net/minecraft/client/1.19-20220607.102129/client-1.19-20220607.102129-srg.jar%23173!/data/.mcassetsroot' uses unexpected schema
[15Jul2022 09:45:53.175] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[15Jul2022 09:45:53.707] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Setting user: krjencik
[15Jul2022 09:45:53.887] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Backend library: LWJGL version 3.3.1 build 7
[15Jul2022 09:45:55.218] [modloading-worker-0/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to register automatic subscribers. ModID: armorpointspp, class dev.cheos.armorpointspp.Armorpointspp
java.lang.NoClassDefFoundError: net/minecraftforge/client/event/ClientPlayerNetworkEvent$LoggedInEvent
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredMethods(Class.java:3402) ~[?:?]
	at java.lang.Class.privateGetPublicMethods(Class.java:3427) ~[?:?]
	at java.lang.Class.getMethods(Class.java:2019) ~[?:?]
	at net.minecraftforge.eventbus.EventBus.registerClass(EventBus.java:75) ~[eventbus-6.0.0.jar%2378!/:?]
	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:118) ~[eventbus-6.0.0.jar%2378!/:?]
	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.lambda$inject$6(AutomaticEventSubscriber.java:61) ~[javafmllanguage-1.19-41.0.98.jar%23175!/:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:48) ~[javafmllanguage-1.19-41.0.98.jar%23175!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:78) ~[javafmllanguage-1.19-41.0.98.jar%23175!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:121) ~[fmlcore-1.19-41.0.98.jar%23174!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraftforge.client.event.ClientPlayerNetworkEvent$LoggedInEvent
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:141) ~[securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 18 more
[15Jul2022 09:45:55.573] [modloading-worker-0/INFO] [Dungeon Crawl/]: Here we go! Launching Dungeon Crawl 2.3.11...
[15Jul2022 09:45:55.604] [modloading-worker-0/INFO] [doubleslabs/]: Loaded 0 vertical slab support classes
[15Jul2022 09:45:55.604] [modloading-worker-0/INFO] [doubleslabs/]: Loaded 2 horizontal slab support classes
[15Jul2022 09:45:55.610] [modloading-worker-0/INFO] [doubleslabs/]: Loaded 0 container support classes
[15Jul2022 09:45:55.804] [modloading-worker-0/INFO] [Bo's Player Companions/]: Thanks for using Bo's Player Companions (https://www.curseforge.com/minecraft/mc-mods/player-companions). I hope you enjoy the mod. :)
[15Jul2022 09:45:55.835] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Menu Types ...
[15Jul2022 09:45:55.842] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Entity Types ...
[15Jul2022 09:45:55.853] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Items ...
[15Jul2022 09:45:55.875] [modloading-worker-0/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 41.0.98, for MC 1.19 with MCP 20220607.102129
[15Jul2022 09:45:55.875] [modloading-worker-0/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v41.0.98 Initialized
[15Jul2022 09:45:55.957] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Blocks ...
[15Jul2022 09:45:55.959] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Sound Events ...
[15Jul2022 09:45:55.961] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Biome Modifier Serializers ...
[15Jul2022 09:45:56.061] [modloading-worker-0/INFO] [Bo's Player Companions/]: ? Register Player Companions Common config ...
[15Jul2022 09:45:56.399] [Render thread/FATAL] [net.minecraftforge.fml.ModLoader/LOADING]: Failed to complete lifecycle event CONSTRUCT, 1 errors found
[15Jul2022 09:45:56.584] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.sound.SoundEngineLoadEvent to a broken mod state
[15Jul2022 09:45:56.776] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterColorHandlersEvent$Block to a broken mod state
[15Jul2022 09:45:56.779] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterColorHandlersEvent$Item to a broken mod state
[15Jul2022 09:45:56.977] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RenderLevelStageEvent$RegisterStageEvent to a broken mod state
[15Jul2022 09:45:57.090] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterParticleProvidersEvent to a broken mod state
[15Jul2022 09:45:57.197] [Render thread/INFO] [com.mojang.text2speech.NarratorWindows/]: Narrator library for x64 successfully loaded
[15Jul2022 09:45:57.345] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterClientReloadListenersEvent to a broken mod state
[15Jul2022 09:45:57.346] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$RegisterLayerDefinitions to a broken mod state
[15Jul2022 09:45:57.346] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$RegisterRenderers to a broken mod state
[15Jul2022 09:45:57.348] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterTextureAtlasSpriteLoadersEvent to a broken mod state
[15Jul2022 09:45:57.350] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterClientTooltipComponentFactoriesEvent to a broken mod state
[15Jul2022 09:45:57.352] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterEntitySpectatorShadersEvent to a broken mod state
[15Jul2022 09:45:57.353] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterKeyMappingsEvent to a broken mod state
[15Jul2022 09:45:57.358] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterRecipeBookCategoriesEvent to a broken mod state
[15Jul2022 09:45:57.365] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterGuiOverlaysEvent to a broken mod state
[15Jul2022 09:45:57.373] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterDimensionSpecialEffectsEvent to a broken mod state
[15Jul2022 09:45:57.376] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterNamedRenderTypesEvent to a broken mod state
[15Jul2022 09:45:57.554] [Render thread/INFO] [net.minecraft.server.packs.resources.ReloadableResourceManager/]: Reloading ResourceManager: Default
[15Jul2022 09:45:57.974] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.ModelEvent$RegisterGeometryLoaders to a broken mod state
[15Jul2022 09:45:59.071] [Worker-Main-3/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:45:59.731] [Worker-Main-1/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:01.970] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.ModelEvent$RegisterAdditional to a broken mod state
[15Jul2022 09:46:02.533] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:02.706] [Worker-Main-5/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:03.969] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:03.974] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:03.994] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:04.015] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:04.021] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:04.030] [Worker-Main-2/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Pre to a broken mod state
[15Jul2022 09:46:04.116] [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: minecraft:item.goat_horn.play
[15Jul2022 09:46:04.116] [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[15Jul2022 09:46:04.220] [Render thread/INFO] [com.mojang.blaze3d.audio.Library/]: OpenAL initialized on device OpenAL Soft on Speakers (High Definition Audio Device)
[15Jul2022 09:46:04.222] [Render thread/INFO] [net.minecraft.client.sounds.SoundEngine/SOUNDS]: Sound engine started
[15Jul2022 09:46:04.222] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.sound.SoundEngineLoadEvent to a broken mod state
[15Jul2022 09:46:04.488] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[15Jul2022 09:46:04.736] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.736] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[15Jul2022 09:46:04.739] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.739] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[15Jul2022 09:46:04.747] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.747] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[15Jul2022 09:46:04.754] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.755] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[15Jul2022 09:46:04.757] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.757] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[15Jul2022 09:46:04.761] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:04.761] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[15Jul2022 09:46:04.766] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:05.277] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.ModelEvent$BakingCompleted to a broken mod state
[15Jul2022 09:46:05.461] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.466] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.598] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.601] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.605] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.613] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.614] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.614] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.615] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.616] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.616] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.624] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.626] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.630] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.631] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.638] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.639] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.647] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.647] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.648] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.654] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.655] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$CreateSkullModels to a broken mod state
[15Jul2022 09:46:05.655] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.EntityRenderersEvent$AddLayers to a broken mod state
[15Jul2022 09:46:05.780] [Render thread/WARN] [net.minecraft.client.renderer.ShaderInstance/]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[15Jul2022 09:46:05.912] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.RegisterShadersEvent to a broken mod state
[15Jul2022 09:46:05.939] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[15Jul2022 09:46:05.947] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:05.948] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[15Jul2022 09:46:05.950] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:05.950] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[15Jul2022 09:46:05.951] [Render thread/ERROR] [net.minecraftforge.fml.ModLoader/]: Cowardly refusing to send event net.minecraftforge.client.event.TextureStitchEvent$Post to a broken mod state
[15Jul2022 09:46:06.150] [Render thread/FATAL] [net.minecraftforge.common.ForgeMod/]: Preparing crash report with UUID 1b7e77c5-33ee-4eed-ac02-68ed0280fc96
[15Jul2022 09:46:06.151] [Render thread/FATAL] [net.minecraftforge.client.loading.ClientModLoader/]: Crash report saved to C:\Users\Kr Jen\AppData\Roaming\.minecraft\crash-reports\crash-2022-07-15_09.46.06-fml.txt
[15Jul2022 09:46:06.153] [Render thread/FATAL] [net.minecraftforge.common.ForgeMod/]: Preparing crash report with UUID ee6e15ba-745e-4196-bf30-8eab991a462a
```

**Comment by Cheos137** [comment]
Looks like something in the forge api changed... gonna look into it

**Comment by Cheos137** [comment]
fixed in 3.0.7
